### DISCUSSION TITLE
Disable Quit-Dialog

### DISCUSSION THREAD

**Discussion by JoshKornfeld** [SafeExamBrowser/seb-mac#410] [Q&A]
Hello,

I'm looking at how to disable the quit dialog after pressing the button to quit SEB. I've found the config key 'quitURLConfirm' and also this setting in the exam pane. 
![image](https://github.com/SafeExamBrowser/seb-mac/assets/49129464/441c37f1-f024-4127-9dfc-ce5d34e791aa)
Despite setting the key to false and also ensuring the checkbox is not set, I get a dialog when trying to exit SEB
![image](https://github.com/SafeExamBrowser/seb-mac/assets/49129464/dcb5d56b-e608-414c-b1e5-3cb879f0e42a)
Are you able to disable this dialog? Am I missing something? 

Thanks in advance :)

**Comment by danschlet** [comment]
I don't know if you were aware that this setting Ask user to confirm is only affecting the Quit Link feature (it is in its section, also see manual). Don't know if you tested if the confirmation dialog is displayed only by using the SEB Quit button, there this setting doesn't have any effect.

We think quitting SEB at any point, especially while in an exam, should always be confirmed first. We want to avoid students loosing something they might have entered in an exam question for example.
